### PR TITLE
Style C: Add option for 'Style 2', and update header and body fonts

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds a style pack option for 'Style 2' (Style C in the mockups).

It also enqueues the fonts from that mockup (Montserrat) and sets the header and body fonts to match the design.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`; confirm there are no errors (since the build process has been changed). 
2. Navigate to the Customizer > Style Pack; switch to 'Style 2'.
3. On the front-end, confirm that it's now loading the 'Style 2' fonts -- Montserrat for the headers, and sans-serif for the body fonts:

image

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

File: inc/style-packs.php [hunk 1]
```diff
@@ -16,17 +16,22 @@
 		'styles'             => array(
 			'default' => esc_html__( 'Default Style', 'newspack' ),
 			'style-1' => esc_html__( 'Style 1', 'newspack' ),
+			'style-2' => esc_html__( 'Style 1', 'newspack' ),
```
Comment (jeffersonrabb) [inline inc/style-packs.php]:
```suggestion
			'style-2' => esc_html__( 'Style 2', 'newspack' ),
```

File: sass/styles/style-1/variables-style/_fonts.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
 // Font and typographic variables
 
-$font__body: "IBM Plex Serif", Georgia, serif;
-$font__heading: "IBM Plex Serif", Georgia, serif;
+$font__heading: "Montserrat", 'Helvetica Neue', 'Helvetica', 'Arial Black', 'Arial', sans-serif;
```
Comment (jeffersonrabb) [inline sass/styles/style-1/variables-style/_fonts.scss]: Did you mean to make these changes to the `style-1` fonts? It appears these are identical to `style-2`?

Comment (laurelfulford) [reply sass/styles/style-1/variables-style/_fonts.scss]: Gah, no -- sorry about that! This has been undone in 8fd3d95.

Trying to do too many different things at once 😐 thanks for catching that!

File: inc/style-packs.php (hunk 1 above)
Comment (laurelfulford) [reply inc/style-packs.php]: D'oh! This has been fixed in 4ec470d.

**Comment by laurelfulford** [comment]
Thanks Jeff!

**Pull request merged**